composer.json

```diff
@@ -15,14 +15,16 @@
         }
     },
     "require": {
-        "php": ">=7.0|>=8.0",
-        "doctrine/orm": "^2.0",
+        "php": "^7.1|^8.0",
+        "doctrine/orm": "^2.5",
         "psr/log": "^1.0|^2.0",
-        "symfony/property-access": "^2.8|^3.0|^4.0|^5.0",
-        "ext-json": "*"
+        "ext-json": "*",
+        "symfony/property-access": "^3.0|^4.0|^5.0|^6.0",
+        "doctrine/annotations": "^1.0|^2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0|^9.0"
+        "phpunit/phpunit": "^6.0|^9.0",
```

Comment:
Let's update this dependency. I think that `^7.5 || ^8.5 || ^9.5` should cover our cases.